swagger-backend.yaml

```diff
@@ -0,0 +1,191 @@
+---
+swagger: '2.0'
+info:
+  description: Flotta backend API for remote data storage
+  title: Flotta backend API
+  version: 1.0.0
+schemes:
+  - https
+consumes:
+  - application/json
+produces:
+  - application/json
+basePath: /api/flotta-backend/v1
+definitions:
+  error:
+    type: object
+    required:
+      - message
+    properties:
+      message:
+        type: string
+  device-configuration-response:
+    type: object
+    properties:
+      message:
+        type: string
+      device-configuration:
+        $ref: 'swagger.yaml#/definitions/device-configuration'
+  device-registration-request:
+    type: object
+    properties:
+      status:
+        type: boolean
+  device-registration-response:
+    type: object
+    properties:
+      target-namespace:
+        type: string
+      message:
+        type: string
+paths:
+  '/namespaces/{namespace}/devices/{device-id}/deregister':
```

Comment:
`.../deregistrable` or `.../finalize` (if you agree with [my comment](https://github.com/project-flotta/flotta-operator/pull/236/files/d712fed91ac4c26188efc552de3230ba6123160d#r884143325)

Reply:
I'll take `/finalize` :smile:

Reply:
s/POST/PUT/ I realized that semantically doesn't make sense to do a POST if you haven't done a PUT first. So PUT it is.